packages/bundlers/experimental/src/ExperimentalBundler.js

```diff
@@ -1163,7 +1230,14 @@ function createIdealGraph(
         dependencyBundleGraph.getNodeIdByContentKey(String(mainNodeId)),
       ]);
     }
-
+    let mainReachableRoot = reachableRoots.getNodeIdByContentKey(
```

Comment:
This section fixes the bug in `react-spectrum` (missing content key) 

When merging type bundles, since I moved the step to later on, we need to patch up the other graphs drawing connections to any children to their new parent